docs/hub/datasets-viewer-configure.md

```diff
@@ -0,0 +1,40 @@
+# Configure the Dataset Viewer
+
+The Dataset Viewer supports many data files formats, from text to tabular and from image to audio formats.
+It also separates the train/validation/test splits based on file and folder names.
```

Comment:
Should we give a link + explanation of what "splits" are?

Reply:
I linked to wikipedia